### PR TITLE
Board page

### PR DESCRIPTION
**Pull request by whirligigY**

### CODE REVIEW

File: src/components/BoardTitleTextarea/index.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          onFocus={handleClear}
+        />
+      </Form.Group>
+    </Form>
+  );
+};
+
+export { BoardTitleTextarea };
+
+/*
```
Comment (Taneros) [inline src/components/BoardTitleTextarea/index.jsx:27]: remove commented code

File: src/components/TaskDescription/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  };
+
+  return (
+    <div className="task-description">
+      <h3>Описание задачи</h3>
+      <Form.Control
+        as="textarea"
+        className={`description-text ${textareaActive ? '' : 'hover'}`}
+        rows={textareaActive ? 3 : 2}
+        placeholder="Добавьте описание задачи"
```
Comment (Taneros) [inline src/components/TaskDescription/index.jsx:22]: are we making an english version?

File: src/components/TaskDescription/index.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        onFocus={() => setAreaState(true)}
```
Comment (Taneros) [inline src/components/TaskDescription/index.jsx:23]: placeholder="Insert task description"

File: src/components/BoardTitleTextarea/index.jsx (hunk 1 above)
Comment (Taneros) [reply src/components/BoardTitleTextarea/index.jsx:27]: or maybe can stash this piece of code?

File: src/pages/HomePage/index.jsx [hunk 4]
```diff
@@ -18,14 +18,14 @@ const HomePage = () => {
...
-      <h2>Твои доски</h2>
-      <div>
-        {/*{boards.map((item) => <div></div>)}*/}
-      </div>
-    </Row>
-  </h1>;
+  return (
+    <h1>
+      <Row>
+        <h2>Твои доски</h2>
```
Comment (Taneros) [inline src/pages/HomePage/index.jsx:24]: Change to : <h2>Your boards</h2>

File: src/pages/HomePage/index.jsx [hunk 5]
```diff
@@ -18,14 +18,14 @@ const HomePage = () => {
...
+        <div>{/*{boards.map((item) => <div></div>)}*/}</div>
```
Comment (Taneros) [inline src/pages/HomePage/index.jsx:25]: stash commented code?

File: src/pages/HomePage/index.jsx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from "react";
-import { Row } from 'react-bootstrap';
+import { Row } from "react-bootstrap";
 
 const HomePage = () => {
   // const [boards, setBoards] = useState([]);
```
Comment (Taneros) [inline src/pages/HomePage/index.jsx:5]: stash commented code?

File: src/App.css [hunk 7]
```diff
@@ -1,6 +1,68 @@
+/*
```
Comment (Taneros) [inline src/App.css:1]: stash commented code?

File: package-lock.json [hunk 8]
```diff
@@ -7925,12 +7925,25 @@
       }
     },
     "node_modules/history": {
+<<<<<<< HEAD
```
Comment (Taneros) [inline package-lock.json:7928]: need to resolve this merge issue

File: src/components/TaskDescription/index.jsx (hunk 2 above)
Comment (Taneros) [reply src/components/TaskDescription/index.jsx:22]: placeholder="Add task description"

**Review by Taneros — CHANGES_REQUESTED** [review]

File: src/components/AddButton/index.jsx [hunk 9]
```diff
@@ -1,83 +1,47 @@
...
-    setCondition({ ...condition, [field]: value });
-  }
 
   return (
-    <div>
-      {condition.isFormActive
-        ? renderForm(type, text, styles)
-        : renderButton(type, text)}
+    <div ref={formRef}>
+      {isFormActive ? (
```
Comment (IgorLap239) [inline src/components/AddButton/index.jsx:30]: another code structure is recommended here (by Alexei)
example: 
    {isFormActive && (
        <RenderFormAddButton
          onChangeState={onChangeState}
          handleAddActivity={handleAddActivity}
          placeholder={placeholder}
          input={input}
          textBtn={textBtn}
        />
      )}
{!isFormActive && (
        <RenderAddButton
          type={type}
          textBtn={textBtn}
          onChangeState={onChangeState}
        />
      )}

**Pull request merged**